releases/3.3.0/artifacts.json

```diff
@@ -0,0 +1,242 @@
+{
+  "3.3.0": {
+    "holoscan": {
+      "debian-version": "3.3.0-1",
```

Comment:
Note that the final debian package version tag is subject to change.

Reply:
Will open another PR over the `release/3.3.0` branch.